### ISSUE TITLE
the workflow status is handled hardly, almost being pending. code issue?

### ISSUE THREAD

**Issue by lxia-hub** [kimios/kimios#44]
![image](https://user-images.githubusercontent.com/69227782/108582907-fa6e5c00-7370-11eb-93a5-514f6230b1d7.png)

**Comment by devtom30** [comment]
Please, can you explain what's happening ?

**Comment by lxia-hub** [comment]
yeah. to set this doc a workflow status. it is showing pending. cannot set successfully. and I don't know what happened. you can use the demo to test. it happened mostly.
